### PR TITLE
V1 merge supportv0

### PR DESCRIPTION
**Pull request by drodenberg**
Resolves: attach_gh_issue_here

### PR Safety Checklist:

 - [ ] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [ ] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [ ] (optional) All external API changes have been documented
 - [ ] (optional) Build docs: `npm run docs`

### CODE REVIEW

File: runfile.js [hunk 1]
```diff
@@ -6,17 +6,17 @@ const packageJson = require('./package.json')
...
 
-  // publish with a `shipping` tag so we can control the latest/next tags manually since NPM defaults to latest if no tag is provided
+  // publish with a `shipping` tag so we can control the legacy/next tags manually since NPM defaults to legacy if no tag is provided
   run(`export SHIP=true && npm run build && npm publish -f --tag shipping`)
   // remove the temporary shipping tag
   run(`npm dist-tags remove ${packageName} shipping || true`)
 
-  // only if the latest is less than the current version do we tag it (this ignores support branches)
-  if (semver.gt(packageJson.version, latestVersion)) {
-    // add the appropriate next or latest tag manually to the version just published
```
Comment (drodenberg) [inline runfile.js:19]: revert this code

File: src/LayerPanel/LayerPanelLayersPage/index.js [hunk 2]
```diff
@@ -96,11 +96,14 @@ class LayerPanelLayersPage extends Component {
     if (disableHover) return
     // clear the previously selected feature before adding newly selected feature so only one feature is "selected" at a time
     this.selectInteraction.getFeatures().clear()
-    features.forEach(feature => {
-      if (feature.get('_ol_kit_feature_visibility')) {
-        this.selectInteraction.getFeatures().push(feature)
-      }
-    })
+
+    if (features) {
```
Comment (drodenberg) [inline src/LayerPanel/LayerPanelLayersPage/index.js:100]: add features?.length

File: src/LayerPanel/LayerPanelLayersPage/index.js [hunk 3]
```diff
@@ -393,15 +402,15 @@ class LayerPanelLayersPage extends Component {
             {layerFilter(layers).filter((layer) => {
               const filteredFeatures = this.getFeaturesForLayer(layer)
 
-              return !enableFilter || !(layer instanceof olLayerVector) ? true : filteredFeatures?.length
+              return !enableFilter || !(layer instanceof olLayerVector) || this.props.shouldHideFeatures(layer) ? true : filteredFeatures.length
```
Comment (drodenberg) [inline src/LayerPanel/LayerPanelLayersPage/index.js:405]: add ? to the .length

**Review by AtherBilal — DISMISSED** [review]

**Review by AtherBilal — APPROVED** [review]

File: src/LayerPanel/LayerPanelLayersPage/index.js [hunk 4]
```diff
@@ -273,8 +277,15 @@ class LayerPanelLayersPage extends Component {
 
     if (layerCheckboxClick && layerVisibility === INDETERMINATE) {
       layer.setVisible(true)
+      layer.set('_ol_kit_layerpanel_visibility', true)
+    } else if (layerCheckboxClick) {
+      const lv = !layer.getVisible()
```
Comment (IDisposable) [inline src/LayerPanel/LayerPanelLayersPage/index.js:282]: A better name might be `toggledLayerVisibility` (assuming we're trying not to shadow/change the in-scope `layerVisibility` ... maybe it just shouldn't be `const` so we're not introducing another variable for no particular need?).

File: src/Popup/PopupInsert/PopupActionGroup/index.js [hunk 5]
```diff
@@ -18,8 +18,19 @@ class PopupActionGroup extends Component {
...
 
-  onHover (hover) {
-    this.setState({ showFlyout: hover })
+  onHover (hover, delay = false) {
+    if (delay) {
+      // allow a slight delay to account for mousing over scroll bars
+      // when moving to flyout on certain browsers/OS
+      this.timeout = setTimeout(() => this.setState({ showFlyout: hover }), 50)
+    } else {
+      if (this.timeout) {
```
Comment (IDisposable) [inline src/Popup/PopupInsert/PopupActionGroup/index.js:27]: Would it be better to do this clear-down of `this.timeout` irrespective of the `delay` argument before setting up the timeout or direct setting? It looks wrong in that if you call `onHover(hover, true)` to set up the timeout, and then it gets called again _before_ the timeout expires then we'll leak the original `setTimeout` return. Additionally it seems a little odd/dangerous that calling it once with `true` for delay and then calling it again with `false` for delay is the expected path.

File: src/Popup/PopupInsert/PopupActionGroup/index.js [hunk 6]
```diff
@@ -31,7 +42,7 @@ class PopupActionGroup extends Component {
     return (
       <div style={{ position: 'relative' }} ref={(element) => { this.el = element }}>
         <Container onMouseEnter={() => this.onHover(true)}
```
Comment (IDisposable) [inline src/Popup/PopupInsert/PopupActionGroup/index.js:44]: Seems we should be explicit about the passing of `false` for delay here...

**Pull request merged**